clang/include/clang/CConv/ProgramVar.h

```diff
@@ -153,15 +156,73 @@ class FunctionParamScope : public ProgramVarScope {
     return "FuncParm_" + FName;
   }
 
-  static FunctionParamScope *getFunctionParamScope(std::string FnName,
-                                                   bool IsSt);
+  std::string getFName() const {
```

Comment:
Could this instead return a `const std::string &` or `llvm::StringRef` to avoid copying the string?

Reply:
Agree. Done.